takwimu/takwimu_ui/src/components/Faqs/index.js

```diff
@@ -31,12 +40,12 @@ function Faqs({ classes, faqs, ...props }) {
         direction="column"
         justify="flex-start"
       >
-        {faqs.faqsList.map(faq => (
-          <Faq expandTitle={faq.question} key={faq.question}>
+        {faqs.faqs.map(faq => (
+          <Faq expandTitle={faq.value.question} key={faq.value.question}>
```

Comment:
why `faq: {value : { question, answer }}` instead of just `faq:  { question, answer }`  https://github.com/TakwimuAfrica/Takwimu/pull/511/files#diff-4792da43157978d4313fbab80f3866bfR167

Reply:
Sticking to wagtail freeform/StreaFields format: `block: { value: block }`. Otherwise UI would need to know where a certain value comes from to determine it's structure i.e. should it check for `value` or just consume the it directly.